doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -246,6 +246,11 @@ nRF Desktop
     * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
     * Limited the number of key slots in the PSA Crypto core to 10 (:kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`).
 
+  * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC (the option was enabled by default, because clock control is enabled by the MPSL - Multiprotocol Service Layer).
```

Comment:
```suggestion
  * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC.
    The option was enabled by default, because the clock control is enabled by the Multiprotocol Service Layer (MPSL).
```